model/src/main/java/no/nav/foreldrepenger/vtp/testmodell/repo/impl/BasisdataProviderFileImpl.java

```diff
@@ -68,36 +68,36 @@ public IdentGenerator getIdentGenerator() {
 
     private void loadAdresser() throws IOException {
         try (InputStream is = getClass().getResourceAsStream("/basedata/adresse-maler.json")) {
-            TypeReference<List<AdresseModell>> typeRef = new TypeReference<List<AdresseModell>>() {
+            TypeReference<List<AdresseModell>> typeRef = new TypeReference<>() {
             };
-            List<AdresseModell> adresser = jsonMapper.lagObjectMapper().readValue(is, typeRef);
+            List<AdresseModell> adresser = mapper.readValue(is, typeRef);
```

Comment:
må du gå veien om typereference her, kan du ikke gjøre readValue(is,AdresseModell.class) drekte

Reply:
Byttet til: List<AdresseModell> adresser = Arrays.asList(mapper.readValue(is, AdresseModell[].class));